README.md

```diff
@@ -1,15 +1,15 @@
-:-------------------: | :---------------------------------------------------------------------------------------------------------------------------:  | ---
-**Linux/Python 2.7.12**   | [![TeamCity](http://dev.aegresco.com/teamcity/app/rest/builds/buildType:(id:Csbuild_LinuxPython2)/statusIcon)](http://dev.aegresco.com/teamcity/viewType.html?buildTypeId=Csbuild_LinuxPython2&guest=1)| |
-**Linux/Python 3.5.2**   | [![TeamCity](http://dev.aegresco.com/teamcity/app/rest/builds/buildType:(id:Csbuild_LinuxPython3)/statusIcon)](http://dev.aegresco.com/teamcity/viewType.html?buildTypeId=Csbuild_LinuxPython3&guest=1)| |
-**macOS/Python 2.7.12**   | [![TeamCity](http://dev.aegresco.com/teamcity/app/rest/builds/buildType:(id:Csbuild_MacOSPython2)/statusIcon)](http://dev.aegresco.com/teamcity/viewType.html?buildTypeId=Csbuild_MacOSPython2&guest=1)| |
-**macOS/Python 3.5.2**   | [![TeamCity](http://dev.aegresco.com/teamcity/app/rest/builds/buildType:(id:Csbuild_MacOSPython3)/statusIcon)](http://dev.aegresco.com/teamcity/viewType.html?buildTypeId=Csbuild_MacOSPython3&guest=1)| |
-**Windows/Python 2.7.12** | [![TeamCity](http://dev.aegresco.com/teamcity/app/rest/builds/buildType:(id:Csbuild_WindowsPython27)/statusIcon)](http://dev.aegresco.com/teamcity/viewType.html?buildTypeId=Csbuild_WindowsPython27&guest=1)| |
-**Windows/Python 3.4.4** | [![TeamCity](http://dev.aegresco.com/teamcity/app/rest/builds/buildType:(id:Csbuild_WindowsPython34)/statusIcon)](http://dev.aegresco.com/teamcity/viewType.html?buildTypeId=Csbuild_WindowsPython34&guest=1)| |
-**Windows/Python 3.5.3** | [![TeamCity](http://dev.aegresco.com/teamcity/app/rest/builds/buildType:(id:Csbuild_WindowsPython35)/statusIcon)](http://dev.aegresco.com/teamcity/viewType.html?buildTypeId=Csbuild_WindowsPython35&guest=1)| |
-**Windows/Python 3.6.1** | [![TeamCity](http://dev.aegresco.com/teamcity/app/rest/builds/buildType:(id:Csbuild_WindowsPython36)/statusIcon)](http://dev.aegresco.com/teamcity/viewType.html?buildTypeId=Csbuild_WindowsPython36&guest=1)| |
+**Test Suite**            | **Status (master)** | **Status (develop)** | &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+:-----------------------: | :-----------------: | :------------------: | ---
```

Comment:
It's always had an extra column, I just left it like that.  And that error message if you go to the status widget link directly says "Not found".  Don't know why it's different or not showing up, but it's probably something on the TeamCity side of things and not in the readme.

Reply:
I believe the extra column is there because Gitswarm's markdown parsing was broken. It can be removed. It's not something you caused but you might as well fix it while you're in there.

As for the status icon, change `master` to `<default>`.

Reply:
Didn't need to change the status icon branch name, just needed to re-run the master builds on TeamCity.  They should start showing up now.